font/font.go

```diff
@@ -0,0 +1,13 @@
+package font
+
+import (
+	"github.com/benoitkugler/textlayout/fonts"
+	"github.com/benoitkugler/textlayout/fonts/truetype"
+)
+
+type Font = *truetype.Font
```

Comment:
This naming suggest that we only support truetype/opentype font files.

I seems to me than some people have expressed interest for older font formats. We probably should discuss what formats we want to support, but in the meantime, I suggest renaming to `OpentypeFont` for instance.

Reply:
There certainly is some confusion about Font vs Face which led to me taking this approach.
Going back to just `Face` for now to avoid the implication of only truetype support.